src/main/kotlin/com/dp/advancedgunnerycontrol/utils/WeaponTagUtils.kt

```diff
@@ -103,31 +98,6 @@ fun loadAllTags(ship: FleetMemberAPI) : List<String>{
     return tags.toList()
 }
 
-/**
- * @return approximate angular distance of target from current weapon facing in rad
- * note: approximation works well for small values and is off by a factor of PI/2 for 180°
- * @param entity: Relative coordinates (velocity-compensated)
- */
-fun angularDistanceFromWeapon(entity: Vector2f, weapon: WeaponAPI): Float {
```

Comment:
`angularDistanceFromWeapon`, `linearDistanceFromWeapon` and `determineIfShotWillHit` were duplicated here and in `SpecificAIPluginBase`. I deduplicated the code and moved it to `AGCUitls`. `AGCUitls` already contained simillar geometry related functions, so it seemed a better place than `SpecificAIPluginBase`, which contains tag related stuff.

Reply:
Awesome!